doc/source/ray-core/fault_tolerance/gcs.rst

```diff
@@ -0,0 +1,74 @@
+.. _fault-tolerance-gcs:
+
+GCS Fault Tolerance
+===================
+
+Global Control Service (GCS) is a server that manages cluster-level metadata.
+It also provides a handful of cluster-level operations including :ref:`actor <ray-remote-classes>`, :ref:`placement group <ray-placement-group-doc-ref>` and node management.
+By default, GCS is not fault tolerant since all the data is stored in-memory and its failure means that the entire Ray cluster fails.
```

Comment:
```suggestion
By default, the GCS is not fault tolerant since all the data is stored in-memory and its failure means that the entire Ray cluster fails.
```